mapsforgesrv/src/main/java/com/telemaxx/mapsforgesrv/MapsforgeSrv.java

```diff
@@ -22,102 +22,344 @@
  * 0.16.2: Check for valid tile numbers (JFBeck)
  * 0.16.3: hillshading (JFBeck)
  * 0.16.4: gamma correction (JFBeck)
- * 0.16.5: a lot of improvements/rework (non0303)
  *******************************************************************************/
 
 package com.telemaxx.mapsforgesrv;
 
+import java.io.File;
 import java.net.BindException;
-import java.util.Arrays;
-import java.util.concurrent.LinkedBlockingQueue;
+import java.net.InetSocketAddress;
+import java.util.ArrayList;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
-import org.eclipse.jetty.http2.server.HTTP2CServerConnectionFactory;
-import org.eclipse.jetty.server.HttpConfiguration;
-import org.eclipse.jetty.server.HttpConnectionFactory;
-import org.eclipse.jetty.server.ProxyConnectionFactory;
+import org.apache.commons.cli.*;
 import org.eclipse.jetty.server.Server;
-import org.eclipse.jetty.server.ServerConnector;
-import org.eclipse.jetty.util.thread.ExecutorThreadPool;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-import org.slf4j.bridge.SLF4JBridgeHandler;
 
 public class MapsforgeSrv {
-	
-	private final static String VERSION = "0.16.5"; // starting with eg 0.13, the mapsforge version //$NON-NLS-1$

```

Comment:
0.16.6

Reply:
Version and jar fils is now 0.16.6, includes bugfixes and improvements, sending a PR